### PR TITLE
use prop-types package instead of react's PropTypes

### PR DESCRIPTION
**Pull request by ruddell**
This is needed for the latest version of React Native

### CODE REVIEW

File: templates/ButtonBox.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import React, { PropTypes } from 'react'
+import React from 'react'
 import { TouchableOpacity, Text, Image } from 'react-native'
 import styles from './Styles/ButtonBoxStyles'
+import PropTypes from 'prop-types'
```
Comment (GantMan) [inline templates/ButtonBox.js]: Gonna need to add this package to the install script... it looks like this

```js
  // add prop types
  await ignite.addModule('prop-types', { version: '15.6.0' })
```

And would go along these:  https://github.com/infinitered/ignite-dev-screens/blob/master/plugin.js#L25

**Review by GantMan — CHANGES_REQUESTED** [review]
needs one last thing

File: templates/ButtonBox.js (hunk 1 above)
Comment (ruddell) [reply templates/ButtonBox.js]: I wondered about that.  Added it

**Review by GantMan — APPROVED** [review]
LGTM!

**Comment by GantMan** [comment]
released v2.2.0

**Pull request merged**